src/mini-cast-bar/hooks/index.js

```diff
@@ -0,0 +1,6 @@
+
+export {useMiniPlayerMediaStatusListenerHook} from "./useMiniPlayerMediaStatusListenerHook"
```

Comment:
I know the broad architecture we are following here is the functional components with hooks and for new development, it's often recommended to use functional components with hooks. 

The only concern I have is that both hooks and functional components were introduced in RN 0.59 but the `peerDependencies` that we added in our react-native-wrapper package is 0.41.2. 
1. Should we update our `peerDependencies` to 0.59? 
2. If so, any existing customers who are on <0.59 are forced to update to 0.59, can we validate this by preparing list of customer who are using our react-native wrapper and what version of react-native they are on?

Reply:
Whom should i ask for point 2?

Reply:
Let's go with the following list for now
Tegna - we will get it
NFL > 0.59
NewsON > 0.59